packages/pangea-sdk/pangea/services/prompt_guard.py

```diff
@@ -29,6 +30,27 @@ class PromptGuard(ServiceBase):
 
     service_name = "prompt-guard"
 
+    def __init__(
+        self, token: str, config: PangeaConfig | None = None, logger_name: str = "pangea", config_id: str | None = None
+    ) -> None:
+        """
+        Prompt Guard service client.
+
+        Initializes a new Prompt Guard client.
+
+        Args:
+            token: Pangea API token.
+            config: Configuration.
+            logger_name: Logger name.
+            config_id: Configuration ID.
+
+        Examples:
+            config = PangeaConfig(domain="aws.us.pangea.cloud")
+            prompt_guard = PromptGuard(token="pangea_token", config=config)
```

Comment:
OK, maybe I am missing something obvious, but shouldn't we show where the PromptGuard class coming from, like:

```
from pangea.services.prompt_guard import PromptGuard, Message
```

Reply:
Added now.